pbs_light.go

```diff
@@ -312,7 +311,10 @@ func (deps *auctionDeps) auction(w http.ResponseWriter, r *http.Request, _ httpr
 				CookieFlag:    labels.CookieFlag,
 				AdapterStatus: pbsmetrics.AdapterStatusOK,
 			}
-			bidderLabels[bidder.BidderCode] = &blabels
+			if blables.Adapter == "" {
```

Comment:
eh... it might be more straightforward to just put this in the declaration: `Adapter: openrtb_ext.BidderName(bidder.BidderCode),`

That should have the same effect regardless of whether or not the code is defined in the map.

Reply:
Was worried if we ever have the case where BidderCode is not the same as the cannonical BidderName